### PR TITLE
Fix loading hint invalidate error

### PR DESCRIPTION
**Pull request by shifucun**
The render process terminates for Class node, hence the loading hint erasing should happen before that.

image

### CODE REVIEW

**Comment by pradh** [comment]
A screenshot validation link might be nice, given there are no tests...

File: static/js/browser/kg.js [hunk 1]
```diff
@@ -705,7 +705,14 @@ async function renderKGPage(
   // Add "More" to existing cards.
   util.appendMoreToAll();
 
-  if (NO_POP_TYPES.includes(type)) return;
+  const loadingElem = document.getElementById("page-loading");
```
Comment (beets) [inline static/js/browser/kg.js:708]: nit - move this into the if block since it's not used otherwise

**Comment by pradh** [comment]
LG, and thanks for screenshot and the PR. Defer approval to cyin...

File: static/js/browser/kg.js (hunk 1 above)
Comment (shifucun) [reply static/js/browser/kg.js]: Done

**Review by shifucun — COMMENTED** [review]
> A screenshot validation link might be nice, given there are no tests...

added screenshot

**Review by beets — APPROVED** [review]
thx!

**Pull request merged**